### PR TITLE
Bulk cert gen

### PR DESCRIPTION
**Pull request by junron**
[Task](https://codein.withgoogle.com/dashboard/task-instances/6286183949664256/)

Added display for certificates generated in bulk from CSV. Based on #157 

Demo:
demo

Note: The randomly moving cursor is the selecting the file bit which doesn't show up in the gif as its in another window

### CODE REVIEW

**Comment by junron** [comment]
@nitikadhiman could I have your opinion?

________________________________

**Comment by nikitadhiman** [comment]
This is better. Thanks for your work.

File: src/app/organization/file-upload/file-upload.component.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <div class="file-list">
+      <div *ngFor="let file of files">
+        <span class="file-name">
+          {{file.name}}
+        </span>
+        <mat-progress-bar mode="determinate" [value]="progress[file.name] | async"></mat-progress-bar>
+      </div>
+    </div>
+    <input id="file-upload-select" type="file" #file (change)="uploadFile()" accept=".csv" style="display: none;">
+    <button onclick="document.getElementById('file-upload-select').click()">Select file</button>
```
Comment (nikitadhiman) [inline src/app/organization/file-upload/file-upload.component.html]: (click) would do....no need for onclick and after that getting element and doing.click() on that...use typescript concepts

**Comment by junron** [comment]
So I'll go ahead with using my service?

________________________________

**Comment by nikitadhiman** [comment]
Please make the changes rest is fine

**Comment by junron** [comment]
Ok, than

________________________________

**Comment by junron** [comment]
@nikitadhiman have a look

________________________________

**Comment by junron** [comment]
@nikitadhiman Build now passes

________________________________

**Comment by nikitadhiman** [comment]
Submit your task for review if not done already

**Comment by junron** [comment]
It has been accepted some time ago. I assumed this was for merge? See #157

________________________________

**Comment by junron** [comment]
@nikitadhiman is it good enough for merge?

________________________________

**Comment by nikitadhiman** [comment]
Yes

**Comment by junron** [comment]
Awesome, let me know if more changed are required before merge. @nikitadhiman

________________________________

**Comment by nikitadhiman** [comment]
Sure will let you know

**Comment by nikitadhiman** [comment]
Hi, can you resolve conflicts on this

**Comment by junron** [comment]
@nikitadhiman resolved conflicts. Look forward to working with you in GCI next year!

**Comment by junron** [comment]
Uh wasn't this going to be merged
________________________________

**Pull request closed without merging**